src/app/components/copy-to-clipboard/index.html

```diff
@@ -0,0 +1,40 @@
+<sky-demo-page
+  pageTitle="Copy to clipboard"
+  skyuxModule="SkyClipboardModule"
+  packageName="lib-clipboard"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The copy to clipboard component creates a button on your page that enables users to copy the content to their computer's clipboard.
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties>
+    <sky-demo-page-property
+      propertyName="copyTarget"
+      isOptional="true"
+    >
+      Specifies the content to copy. This property binds data, so you must use brackets to evaluate the template expression: <stache-code>[copyTarget]</stache-code>. Without the brackets, Angular treats the value as a string and initializes the target property with the string value instead of evaluating the data value.
```

Comment:
Same question as in the code block docs about the second and third sentences. We should confirm with a dev that it makes sense to include this info.

Reply:
✔️ checked in with Steve and he suggested keeping it.